client/reader/following-manage/index.jsx

```diff
@@ -96,10 +92,19 @@ class FollowingManage extends Component {
 			debounce( this.resizeSearchBox, 50 )
 		);
 		this.resizeSearchBox();
+
+		// this is a total hack. In React-Virtualized you need to tell a WindowScroller when the things
+		// above it has moved with a call to updatePosision().  Our issue is we don't have a good moment
+		// where we know that the content above the WindowScroller has settled down and so instead the solution
+		// here is to call updatePosition in a regular interval. the call takes about 0.1ms from empirical testing.
+		this.updatePosition = setInterval( () => {
+			this.windowScrollerRef && this.windowScrollerRef.updatePosition();
+		}, 100 );
```

Comment:
also, just to be sure, i tested this out with 10x CPU slowdown and 3g internet speeds.
It took ~0.7ms (as opposed to the 0.1ms) which is still acceptable in my mind as its only happening every few hundred ms.